examples/kubernetes/nfs-provisioner/README.md

```diff
@@ -0,0 +1,15 @@
+# Set up a NFS Server on a Kubernetes cluster
+
+To create a NFS provisioner on your Kubernetes cluster, run the following command
+
+```bash
+kubectl create -f https://raw.githubusercontent.com/kubernetes-csi/nfs-driver-smb/master/examples/kubernetes/nfs-provisioner/nfs-server.yaml
```

Comment:
This is pointing to smb repo. Should it point to this repo?

Reply:
Oops, that's a typo